programmers/3일차/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,24 @@
+function solution (numbers) {
+    let arrAllNumber = [];
+    let count = 1;
+
+    for ( let i = 0; i<numbers.length; i++ ) {
```

Comment:
```suggestion
    for ( let i = 0; i < numbers.length; i++ ) {
```
비교연산자 역시  연산자이기에  앞 뒤 공백 필수 입니다 ~~